src/tsdf/read_binary.py

```diff
@@ -6,10 +6,30 @@
 
 import os
 import numpy as np
+import pandas as pd
 from tsdf import numpy_utils 
 from tsdf import tsdfmetadata
 
 
+def load_binaries_to_dataframe(metadatas: '[tsdfmetadata.TSDFMetadata]') -> pd.DataFrame:
+    """
+    Load binary files associated with TSDF and return a combined pandas DataFrame.
+
+    :param metadatas: list of TSDFMetadata objects.
+
+    :return: pandas DataFrame containing the combined data.
+    """
+    # Load the data
+    dataFrames = []
+    for metadata in metadatas:
+        data = load_binary_from_metadata(metadata)
+        df = pd.DataFrame(data, columns=metadata.channels)
+        dataFrames.append(df)
+
+    # Merge the data
+    return pd.concat(dataFrames, axis=1)
```

Comment:
Here we assume that the binary files have no overlapping channels. What would happen in case of having 2 binary files that track the same sensor data over multiple weeks, e.g., week_1_imu.bin and week_2_imu.bin?

Reply:
This is only meant to combine binaries referenced from the same metadata, concatenating them column-wise. We don't really support concatenating them in the time/row direction (do we?), but maybe we should.